### PR TITLE
add synonym "hindlimb zeugopod skin" to "lower leg skin" UBERON:0004264

### LINKED ISSUE #3340: add synonym "hindlimb zeugopod skin" to "lower leg skin" UBERON:0004264
**Uberon term**
"lower leg skin" UBERON:0004264

**Synonym to be added**
hindlimb zeugopod skin

**broad/exact/narrow/related ?**
exact

Needed to describe cell lines in Cellosaurus originated in PMID:11943340
Many thanks
https://orcid.org/0000-0002-0819-0473

### PR DESCRIPTION
**Pull request by paulacalipho**
fixes #3340

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -84444,6 +84444,7 @@ name: lower leg skin
 def: "A zone of skin that is part of a lower leg [Automatically generated definition]." [OBOL:automatic]
 synonym: "hind limb middle limb segment skin" EXACT [OBOL:automatic]
 synonym: "hind limb zeugopod skin" EXACT [OBOL:automatic]
+synonym: "hindlimb zeugopod skin" EXACT [PMID:11943340]
```
Comment (aleixpuigb) [inline src/ontology/uberon-edit.obo:84447]: Thank you Paula, I can't see the usage of this synonym in the reference you have provided. Could it be the wrong PMID?

Comment (paulacalipho) [reply src/ontology/uberon-edit.obo:84447]: @aleixpuigb thank you Aleix for reviewing. Yes, you are right. We just realized that hindlimb zeugopod skin was missing while integrating in Cellosauurs cell lines described in PMID:11943340 (but the term is not described in that paper). I will remove this reference but I do not have any other to add. Should I add my ORCID or leave it empty? Sorry for the confusion

Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:84447]: I think adding your ORCID is enough in this case, thank you!

**Review by aleixpuigb — APPROVED** [review]

**Comment by paulacalipho** [comment]
@aleixpuigb many thanks!

**Pull request merged**